client/src/pages/database/index.tsx

```diff
@@ -27,12 +45,35 @@ export const Database = () => {
   }
 
   const { database } = data;
+  const { apps } = appsData;
 
   if (!database) {
     // TODO nice 404
     return <p>Database not found.</p>;
   }
 
+  const appOptions = apps.map((app) => {
+    return {
+      value: { name: app.name, id: app.id },
+      label: app.name,
+    };
+  });
+
+  const handleConnect = async (databaseId: string, appId: string) => {
+    try {
+      await linkDatabaseMutation({
```

Comment:
do we want to show some kind of feedback that this worked at the end? can be a TODO TOASTIFY :p

Reply:
@pradel Yes, that's a great idea. TODO added.